### PR TITLE
Bug fix: resource_pool utilization in proteinSolvation now works

### PR DESCRIPTION
**Pull request by david451lima**
- Corrected an issue with the charge_computation command;
- If the user specified a resour_pool_id the code wouldn't work, since it wasn't passing the resource_pool_id to the super.simulate methods.

There are some things (change in requirements and __init__ file) that do not relate to this bug fix; it's something that will be used in the next PR (I splitted the changes to have a more manageable PR, but some things were ill divided)

### CODE REVIEW

File: inductiva/molecules/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #pylint: disable=missing-module-docstring
 from .simulators import GROMACS
-from .scenarios import protein_solvation
+from .scenarios import protein_solvation, _post_processing
```
Comment (IvanPombo) [inline inductiva/molecules/__init__.py:3]: I think this part is on a next PR?

File: inductiva/templates/protein_solvation/gromacs/charge_computation.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [{
     "cmd":
-        "gmx pdb2gmx -f protein.pdb -o protein.gro -water tip3p -ff amber99sb-ildn",
+        "gmx pdb2gmx -f 1AKI_clean.pdb -o protein.gro -water tip3p -ff amber99sb-ildn",
```
Comment (IvanPombo) [inline inductiva/templates/protein_solvation/gromacs/charge_computation.json:3]: This is extra in here, no?

File: inductiva/templates/protein_solvation/gromacs/charge_computation.json.jinja [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[{
```
Comment (IvanPombo) [inline inductiva/templates/protein_solvation/gromacs/charge_computation.json.jinja:1]: I think we should follow the approach below of uniformization when calling the protein, but this is ok for now!

File: requirements.txt [hunk 4]
```diff
@@ -18,3 +18,5 @@ imageio-ffmpeg
 ipython
 websockets
 typing-extensions
+MDAnalysis
```
Comment (IvanPombo) [inline requirements.txt:21]: This is of a next PR.

**Review by IvanPombo — APPROVED** [review]
LGTM! Notice that you have here updates for your next PR on viz.

File: inductiva/molecules/__init__.py [hunk 5]
```diff
@@ -1,3 +1,3 @@
 #pylint: disable=missing-module-docstring
 from .simulators import GROMACS
-from .scenarios import protein_solvation
+from .scenarios import protein_solvation  #, _post_processing
```
Comment (fabiocruz) [inline inductiva/molecules/__init__.py:3]: This comment was left in the code.

**Review by fabiocruz — COMMENTED** [review]
Nice catch, @david451lima! Left one minor comment only.

**Pull request merged**